php/libraries/DicomTar.class.inc

```diff
@@ -121,10 +121,10 @@ class DicomTar
                     intval($row['id']),
                     $row['seriesdescription'],
                     intval($row['seriesnumber']),
-                    $row['echotime'],
-                    $row['repetitiontime'],
-                    $row['inversiontime'],
-                    $row['slicethickness'],
+                    \Utility::nullableStringToInt($row['echotime']),
```

Comment:
ah, that answers my earlier question 🤣

Reply:
Whoops, that is actually a bug since those values should be floats, fixed.